tests/wake-format/disallowed-input/stdout

```diff
@@ -24,7 +22,6 @@ def t12 =
     # comment
     )
 
-
 # trailing comment
 
 
```

Comment:
Looks like comments trigger a different pathway for adding the newline; it's likely whatever's going on with the topic/target spacing I pointed out would also need to be addressed separately if either of those happened at the end of the file.

Reply:
Right, comments have a slightly different route, but it should just work for everything besides trailing comments.

Anything in this test case are explicitly not supported by the formatter. In this case it means that comments are not allowed to be the last thing in a file. (or at the very least, you can have it but it'll be formatted wrong)

Reply:
yep can do